views/financial/income.pug

```diff
@@ -29,7 +29,7 @@ block content
             th(colspan="2").breakdown-table__subheading #{__(`Detailed breakdown of income earned in ${year}`)}
           each income in incomeSources
             tr
-              td #{__(`Employer ${count} Total Income`)}
+              td #{__(`${income.name} Total Income`)}
```

Comment:
Actually, let's only translate the string "Total Income" instead of including the employer name